dwl.c

```diff
@@ -1606,7 +1611,7 @@ pointerfocus(Client *c, struct wlr_surface *surface, double sx, double sy,
 		focusclient(c, 0);
 
 	/* If surface is NULL, clear pointer focus */
-	if (!surface) {
+	if (!surface || exclusive_focus != surface) {
```

Comment:
@fauxmight can you revert this line and tell me if it works?

Reply:
No noticeable change.